js/navbar.js

```diff
@@ -0,0 +1,8 @@
+function changeCSS() {
+  const elementToManipulate = document.getElementById("topNavBar");
+  if (elementToManipulate.className === "topnav") {
+    elementToManipulate.className += " responsive";
```

Comment:
Nit: Space should be it's own character, so it's clear that it was put on purpose
```js
elementToManipulate.className += " " + "responsive";
```

(Thank god we have string interpolation in modern ways)